src/components/carousel/index.stories.mdx

```diff
@@ -681,3 +683,84 @@ const Feature = () => (
 		</Carousel>
 	</Story>
 </Canvas>
+
+** Indicators none **
+
+<Canvas>
+	<Story name="Indicators none">
+		<Carousel
+			id="carousel-1"
+			label="Carousel of Images"
+			slidesToShow={1}
+			showLabel
+			indicators="none"
+		>
+			<Carousel.Item label="Slide 1 of 3">
+				<a href="/">
+					<Image src="/computer-user.jpeg" alt="A person sat down using a laptop computer" />
+				</a>
+			</Carousel.Item>
+			<Carousel.Item label="Slide 2 of 3">
+				<Image src="/camera.jpeg" alt="A camera with photos in front of it" />
+			</Carousel.Item>
+			<Carousel.Item label="Slide 3 of 3">
+				<Image src="/canyon.jpeg" alt="Landscape view of a canyon" />
+			</Carousel.Item>
+		</Carousel>
+	</Story>
+</Canvas>
+
+** Indicators dots **
+
+<Canvas>
+	<Story name="Indicators dots">
+		<Carousel
+			id="carousel-1"
+			label="Carousel of Images"
+			slidesToShow={1}
+			showLabel
+			indicators="dots"
+		>
+			<Carousel.Item label="Slide 1 of 3">
+				<Image src="/computer-user.jpeg" alt="A person sat down using a laptop computer" />
+			</Carousel.Item>
+			<Carousel.Item label="Slide 2 of 3">
+				<Image src="/camera.jpeg" alt="A camera with photos in front of it" />
+			</Carousel.Item>
+			<Carousel.Item label="Slide 3 of 3">
+				<Image src="/canyon.jpeg" alt="Landscape view of a canyon" />
+			</Carousel.Item>
+		</Carousel>
+	</Story>
+</Canvas>
+
+** Indicators thumbnails **
```

Comment:
As the thumbnail indicator has scroll into view logic - should we add a story with a lot more images to be able to see this working?

Reply:
Yeah, good idea. I suppose I was shrinking the window to see the effect

Reply:
updated